ci/run_premerge_gpu.sh

```diff
@@ -15,15 +15,57 @@
 # limitations under the License.
 #
 
+# Argument(s):
+#   BUILD_TYPE:   all/specific_test_name, tests to execute
+
 set -ex
+BUILD_TYPE=all
+
+if [[ $# -eq 1 ]]; then
+    BUILD_TYPE=$1
+
+elif [[ $# -gt 1 ]]; then
+    echo "ERROR: too many parameters are provided"
+    exit 1
+fi
+
+init_pipenv() {
+    echo "initializing pip environment: $1"
+    pipenv install -r $1
+    export PYTHONPATH=$PWD
+}
+
+remove_pipenv() {
+    echo "removing pip environment"
+    pipenv --rm
+    rm Pipfile Pipfile.lock
+}
+
+integration_test() {
+    echo "Run integration test..."
+    init_pipenv requirements-dev.txt
+    testFolder="tests/integration_test"
+    rm -rf /tmp/snapshot-storage
+    pushd ${testFolder}
+    pipenv run ./run_integration_tests.sh
+    popd
+    rm -rf /tmp/snapshot-storage
+}
```

Comment:
please also add remove_pipenv as a good context patten to follow even there is only one test in this file for now

Reply:
added remove_pipenv in the end